src/solvers/umfpack.jl

```diff
@@ -475,26 +489,37 @@ function show(io::IO, mime::MIME{Symbol("text/plain")}, F::UmfpackLU)
     end
 end
 
+function serialize(s::AbstractSerializer, L::UmfpackLU{Tv, Ti}) where {Tv, Ti}
+    # TODO: If we can get a C FILE handle we can serialize umfpack_numeric and
```

Comment:
Probably better to ask for a version that serializes to memory, rather than trying to emulate FILE.

Reply:
Ah, you already said that, carry on.